settings.py

```diff
@@ -0,0 +1,110 @@
+import logging
```

Comment:
Is `logging` used for some purposes?

Reply:
Yes, for logging settings (75, 76 lines )

    # LOGGING SETTINGS
    LOG_LEVEL: int = Field(default=logging.WARNING)
    LOG_USE_COLORS: bool = Field(default=False)

Reply:
Let logging settings, or remove them?

Reply:
remain, it's ok